inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -706,13 +706,17 @@ std::shared_ptr<ngraph::Node> V10Parser::XmlDeserializer::createNode(
     auto opsetIt = opsets.find(params.version);
 
     // Try to create operation from loaded opsets
-    static const std::unordered_set<std::string> experimental_detectrons = {"ExperimentalDetectronDetectionOutput",
-                                                                            "ExperimentalDetectronGenerateProposalsSingleImage",
-                                                                            "ExperimentalDetectronPriorGridGenerator",
-                                                                            "ExperimentalDetectronROIFeatureExtractor",
-                                                                            "ExperimentalDetectronTopKROIs"};
-
-    if (experimental_detectrons.count(params.type)) {
+    static const std::unordered_set<std::string> experimental_ops_added_to_opset = {
+            "ExperimentalDetectronDetectionOutput",
+            "ExperimentalDetectronGenerateProposalsSingleImage",
+            "ExperimentalDetectronPriorGridGenerator",
+            "ExperimentalDetectronROIFeatureExtractor",
+            "ExperimentalDetectronTopKROIs",
+            "GRUCell",
+            "RNNCell",
+            "Proposal"};
```

Comment:
Am I right that MO does not generate IRs with `SparseToDense`, `SimplerNMS` and `Unique`, so we can safely drop them and not include to this list?